### PR TITLE
Change time unit of streaming back to millisecond

### PR DESCRIPTION
**Pull request by qintao1976**
In branch-1.6, time unit is millisecond. To avoid breaking change, change the time unit of back to millisecond in master branch to align with branch-1.6.

### CODE REVIEW

File: csharp/Samples/Microsoft.Spark.CSharp/DStreamStateSample.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ internal static void DStreamMapWithStateSample()
                 () =>
                 {
                     SparkContext sc = SparkCLRSamples.SparkContext;
-                    StreamingContext context = new StreamingContext(sc, 10);
+                    StreamingContext context = new StreamingContext(sc, 10000);
```
Comment (skaarthik) [inline csharp/Samples/Microsoft.Spark.CSharp/DStreamStateSample.cs]: add code comments to indicate the param is in millis

File: csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs [hunk 2]
```diff
@@ -295,11 +295,10 @@ private static void DStreamReduceByKeyAndWindowSample()
         [Sample("experimental")]
         internal static void DStreamCSharpInputSample()
         {
-            const int bacthInterval = 2;
             const int numPartitions = 5;
 
             var sc = SparkCLRSamples.SparkContext;
-            var ssc = new StreamingContext(sc, bacthInterval);
+            var ssc = new StreamingContext(sc, 2000);
```
Comment (skaarthik) [inline csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs]: add code comments to indicate the param is in millis

File: csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs [hunk 3]
```diff
@@ -244,7 +244,7 @@ private static void DStreamReduceByKeyAndWindowSample()
         {
             count = 0;
 
-            const int bacthInterval = 2;
+            const long bacthInterval = 2000;
```
Comment (skaarthik) [inline csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs]: consider renaming the variable to batchIntervalInMilliSeconds or something to that effect

File: csharp/Adapter/Microsoft.Spark.CSharp/Streaming/StreamingContext.cs [hunk 4]
```diff
@@ -56,11 +56,11 @@ private StreamingContext(IStreamingContextProxy streamingContextProxy)
         /// Initializes a new instance of StreamingContext with a existing SparkContext 
         /// </summary>
         /// <param name="sparkContext">An existing SparkContext</param>
-        /// <param name="durationSeconds">the time interval at which streaming data will be divided into batches</param>
-        public StreamingContext(SparkContext sparkContext, int durationSeconds)
+        /// <param name="durationMs">the time interval at which streaming data will be divided into batches</param>
```
Comment (skaarthik) [inline csharp/Adapter/Microsoft.Spark.CSharp/Streaming/StreamingContext.cs]: update comments on the param to indicate the unit is in milliseconds

File: csharp/Adapter/Microsoft.Spark.CSharp/Streaming/StreamingContext.cs [hunk 5]
```diff
@@ -106,10 +106,10 @@ public void Stop()
         /// collection. This method allows the developer to specify how long to remember the RDDs (
         /// if the developer wishes to query old data outside the DStream computation).
         /// </summary>
-        /// <param name="durationSeconds">Minimum duration that each DStream should remember its RDDs</param>
-        public void Remember(int durationSeconds)
+        /// <param name="durationMs">Minimum duration that each DStream should remember its RDDs</param>
```
Comment (skaarthik) [inline csharp/Adapter/Microsoft.Spark.CSharp/Streaming/StreamingContext.cs]: add unit to the param comments

File: csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs [hunk 6]
```diff
@@ -69,7 +69,7 @@ internal static void DStreamTextFileSample()
                 () =>
                 {
 
-                    StreamingContext context = new StreamingContext(sc, 2);
+                    StreamingContext context = new StreamingContext(sc, 2000);
```
Comment (skaarthik) [inline csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs]: consider switching to using the batchInterval variable below or adding comments on the unit of the second param in this sample and others below

**Comment by skaarthik** [comment]
LGTM

**Comment by qintao1976** [comment]
@skaarthik I have addressed all your review comments. Thanks

**Pull request merged**